dev/release/verify-release-candidate.sh

```diff
@@ -148,9 +144,6 @@ verify_dir_artifact_signatures() {
     # basename of the artifact
     pushd $(dirname $artifact) >/dev/null
     base_artifact=$(basename $artifact)
-    if [ -f $base_artifact.sha256 ]; then
-      ${sha256_verify} $base_artifact.sha256 || exit 1
-    fi
     if [ -f $base_artifact.sha512 ]; then
       ${sha512_verify} $base_artifact.sha512 || exit 1
     fi
```

Comment:
Can we require sha512 file?

```suggestion
    ${sha512_verify} $base_artifact.sha512 || exit 1
```